<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QFrame" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>884</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1090</width>
     <height>861</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="Devices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>761</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Main">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QLabel" name="main_logo_label">
      <property name="geometry">
       <rect>
        <x>844</x>
        <y>570</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="motor_ui_pb">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>190</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>motor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="inverter_ui_pb">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>inverter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plan_ui_pb">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>190</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>structure</string>
      </property>
     </widget>
     <widget class="QPushButton" name="spd_meter_ui_pb">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>speed meter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="acc_meter_ui_pb">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>330</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Acc meter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gnd_center_ui_pb">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ground center</string>
      </property>
     </widget>
     <widget class="Line" name="gnd_inv_line">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>340</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="inv_motor_line">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="motor_plan_line">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>210</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="acc_cnt_2_line">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>490</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="inv_cnt_2_line">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>490</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="inv_cnt_line">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>250</y>
        <width>20</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="plan_acc_line">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>251</y>
        <width>20</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="controller_ui_pb">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>470</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>controller</string>
      </property>
     </widget>
     <widget class="Line" name="motor_spd_line">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>251</y>
        <width>20</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="spd_cnt_line">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>391</y>
        <width>20</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="acc_cnt_line">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>391</y>
        <width>20</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="close_pb">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>0</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Setting">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>281</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_10">
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="horizontalSpacing">
        <number>7</number>
       </property>
       <property name="verticalSpacing">
        <number>7</number>
       </property>
       <item row="1" column="1">
        <widget class="QCheckBox" name="SendDataPrintFlag">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="DebugPrintFlag">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DebugPrint</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="setting_logo_label">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>540</y>
        <width>190</width>
        <height>190</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="quotex_username_lineEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>250</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLabel" name="quotex_username_label">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>80</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>quotex user name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="quotex_password_label">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>110</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>quotex password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="quotex_password_lineEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>250</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="set_profile_pb">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>set profile</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="main_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>1041</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Siemens Serif SC</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>AvidMech</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_service_pb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>790</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Stop Service</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_service_pb">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>790</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start Service</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
